src/utils/HypervBoxUtil.js

```diff
@@ -31,19 +28,19 @@ var HypervBox = {
   */
 
   hasAdminRights: function() {
-    return util.execFile([this.command(), 'Get-VMHostSupportedVersion']).then(stdout => {
+    return util.execFile([this.command(), 'Get-VMHost']).then(stdout => {
       console.log('stdout: ', stdout);
       // To execute the above command you'll need Admin or Hyper-V-Admin rights.
-      if ( stdout.toUpperCase().indexOf('TRUE') !== -1) {
+      if (stdout.toLowerCase().indexOf('invalidoperation') === -1) {
         return Promise.resolve(true);
       }
+      return Promise.resolve(false);
     }).catch(() => {
       return Promise.resolve(false);
     });
   },
   switchName: function (name) {
-//    return util.execFile([this.command(), '$(Get-VMSwitch | where {$_.SwitchType -eq "external"}).name']).then(out => {
-    return util.execFile([this.command(), '$(Get-VMSwitch).name']).then(out => {
+    return util.execFile([this.command(), '$(Get-VMSwitch | where {$_.SwitchType -eq "external"}).name']).then(out => {
```

Comment:
@pmario this was removed in the latest version of code, after extensive testing on several machines.